main.js

```diff
@@ -19,6 +19,14 @@ async function getImageName(distribution) {
     const tag = getImageTag("", distribution)
     for (const image of ["debian", "ubuntu"]) {
         try {
+            core.startGroup("Care about skopeo")
+            await exec.exec("sudo", [
```

Comment:
Should we also perform apt update?

Reply:
This would make things safer, but also makes the runtime much longer. I'd love to have checked if the command is installed, if not, we should update apt first and then install the package.

Reply:
You can use
https://github.com/dawidd6/action-debian-package/tree/master/node_modules/%40actions/io#which

Reply:
Okay ... I updated the PR with 2754bd6, which looks promising ... please have a alook.